specs/protocol/configurability.md

```diff
@@ -30,47 +31,70 @@ There are four categories of OP Stack configuration options:
-| [L2 block time](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/L2OutputOracle.sol#L105)                         | Frequency with which blocks are produced as a result of derivation.                                                          | [L1 Proxy Admin](#admin-roles)                      |
-| [Resource config](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L338-L340)                       | Config for the EIP-1559 based curve used for the deposit gas market.                                                         | [System Config Owner](#admin-roles)                 |
-| Sequencing window                     | Maximum allowed batch submission gap, after which L1 fallback is triggered in derivation.                                    | Static |
-| [Start block](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L184)                           | Block at which the system config was initialized the first time.                                                             | [L1 Proxy Admin](#admin-roles)                      |
-| Superchain target                     | Choice of cross-L2 configuration. May be omitted in isolated OP Stack deployments. Includes SuperchainConfig and ProtocolVersions contract addresses. | Static |
+| Config Property                       | Description                                                                                                                  | Administrator                       | Standard Config Requirement | Notes |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|-------------------------------------|-------------------------------------|
+| [Batch Inbox address](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L176)                   | L1 address where calldata/blobs are posted (see [Batcher Transaction](../glossary.md#batcher-transaction)).                  | Static | Current convention is `0xff000...000{chainId}`. This doesn't work when `chainId` is `uint256`. | Follow convention if possible. A future hardfork plans to migrate Batch Inbox Addresses to better support `uint256` sized chain IDs. |
+| [Batcher Hash](./system_config.md#batcherhash-bytes32) | A versioned hash of the current authorized batcher sender(s). | [System Config Owner](#admin-roles) | `bytes32(uint256(uint160(batchSubmitterAddress)))` | [Batch Submitter](../protocol/batcher.md) address padded with zeros to fit 32 bytes. |
```

Comment:
I dont follow how `bytes32(uint256(uint160(batchSubmitterAddress)))` makes it standard config?

Reply:
Isn't that the definition of the current versioned hash of the batcher, since it's version 0? https://github.com/ethereum-optimism/specs/pull/134#pullrequestreview-2007307306